product/personas/readme.md

```diff
@@ -0,0 +1,49 @@
+# Personas
```

Comment:
Nitpick: Can we rename this file to README.md for consistency?

Reply:
Done